targets/TARGET_Realtek/TARGET_AMEBA/rtw_emac.cpp

```diff
@@ -32,39 +33,42 @@
 
 #define RTW_EMAC_MTU_SIZE  (1500U)
 
-static emac_interface_t *_emac;
-static emac_link_input_fn link_input_cb;
-static emac_link_state_change_fn link_state_cb;
-static void *link_input_data;
-static void *link_state_data;
+RTW_EMAC::RTW_EMAC() 
+{
+    set_callback_func((emac_callback)(&RTW_EMAC::wlan_emac_recv), this);
+}
 
-static uint32_t wlan_get_mtu_size(emac_interface_t *emac)
+uint32_t RTW_EMAC::get_mtu_size() const
 {
     return RTW_EMAC_MTU_SIZE;
 }
 
-static void wlan_get_ifname(emac_interface_t *emac, char *name, uint8_t size)
+uint32_t RTW_EMAC::get_align_preference() const
+{
```

Comment:
This needs to return something. `1` is fine.

Reply:
Thanks. Confirmed to add return true; .